README.md

```diff
@@ -29,6 +29,41 @@ Tutorials in the **Intro-Full** directory cover
 
 All the tutorial folders can be built using either the `Makefile` or the CMake `CMakeLists.txt` file in each folder.
 
+## CMake
+
+CMake can build against an installed Kokkos library or download one automatically using `FetchContent`.
+
+Without any Kokkos already installed, from an exercise directory, one can run the following:
+
+```shell
+cmake -B build_dir # -DKokkos_* options
+cmake --build build_dir
+```
+
+Kokkos options are described in [CMake options](https://kokkos.org/kokkos-core-wiki/keywords.html).
+
+For example, OpenMP CPU exercises can be built as:
+```shell
+cmake -B build_openmp -DKokkos_ARCH_NATIVE=ON -DKokkos_ENABLE_OPENMP=ON
+cmake --build build_openmp
+```
+
+On Mac, if OpenMP is not available, one can use the Threads backend:
+```shell
+cmake -B build_threads -DKokkos_ARCH_NATIVE=ON -DKokkos_ENABLE_THREADS=ON
+cmake --build build_threads
+```
+
+For a NVIDIA V100 gpu:
+
+```shell
+cmake -B build_cuda -DKokkos_ARCH_NATIVE=ON Kokkos_ARCH_VOLTA70=ON -DKokkos_ENABLE_CUDA=ON
```

Comment:
I can't help thinking that if "native" works then we are likely not cross-compiling and the GPU arch can be auto detected.

Reply:
I think that is a good idea to be as simple as possible. I was a little annoyed to target a specific GPU architecture.
I do not have a Nvidia GPU at hand to see if it is working with Kokkos 4.0.01.